numba/analysis.py

```diff
@@ -1,2 +1,12 @@
-from numba.core.analysis import * # noqa: F403, F401
-from numba.core.analysis import _use_defs_result # noqa: F401
+from numba.core.errors import (deprecate_moved_module,
+                               deprecate_moved_module_getattr)
+from numba.core.utils import PYVERSION
+
+if PYVERSION > (3, 6):
+    __getattr__ = deprecate_moved_module_getattr(__name__,
+                                                 'numba.core.analysis')
+else:
+    from numba.core.analysis import * # noqa: F403, F401
+    from numba.core.analysis import _use_defs_result # noqa: F401
```

Comment:
PEP8 wants two spaces
```suggestion
    from numba.core.analysis import *  # noqa: F403, F401
    from numba.core.analysis import _use_defs_result  # noqa: F401
```